python/ray/_private/worker.py

```diff
@@ -2708,14 +2708,14 @@ def put(
 @PublicAPI
 @client_mode_hook
 def wait(
-    ray_waitables: List[Union["ObjectRef[R]", "ObjectRefGenerator[R]"]],
+    ray_waitables: Union[ObjectRef, ObjectRefGenerator],
```

Comment:
This should be a list.

Why changing this? Seems unrelated to broken referneces

Reply:
Sorry, my mistake - I'll make this a list again.

> Why changing this? Seems unrelated to broken referneces

`sphinx.ext.autodoc` uses type annotations to document parameters and return types for functions.